content/manager/upload-blueprint.md

```diff
@@ -16,7 +16,7 @@ From the Cloudify command-line interface, you can upload your blueprint to Cloud
 
 The following is an example of `upload`:
 {{< gsHighlight  bash >}}
-$ cfy blueprints upload -b BLUEPRINT_ID -p BLUEPRINT_FILE_LOCATION
+$ cfy blueprints upload -b BLUEPRINT_ID -n BLUEPRINT_FILE_LOCATION URL_TO_BLUEPRINT_ARCHIVE
```

Comment:
The explanation before this says "If you have downloaded cloudify-nodecellar-example from github..." so it looks like we just have to remove the -n from the command because it assumes that you already downloaded it to the manager.

What do you think?